apps/ios/Shared/Views/UserSettings/PrivacySettings.swift

```diff
@@ -68,6 +68,25 @@ struct PrivacySettings: View {
                         Text("Opening the link in the browser may reduce connection privacy and security. Untrusted SimpleX links will be red.")
                     }
                 }
+
+                Section {
+                    settingsRow("person") {
+                        Toggle("Contacts", isOn: $useLinkPreviews)
+                    }
+                    settingsRow("person.2") {
+                        Toggle("Small groups (max 10)", isOn: Binding.constant(false))
+                    }
```

Comment:
\+ "Incognito contacts and small groups"
or "Incognito contacts, Incognito small groups separately"

Reply:
I decided not to differentiate it initially